.github/workflows/docker-build-test.yaml

```diff
@@ -196,7 +196,7 @@ jobs:
     with:
       GIT_SHA: ${{ needs.determine-docker-build-metadata.outputs.gitSha }}
       FORGE_TEST_SUITE: compat
-      IMAGE_TAG: 843b204dce971d98449b82624f4f684c7a18b991 # test against the latest build on testnet branch
+      IMAGE_TAG: 827e1c63739bbf06dad94f503a325e08b0daeb97 # test against the latest build on testnet branch
```

Comment:
This change will take effect immediately which means that anything that runs with a merge base oilder than this will immediately fail with the gas error! lets wait a couple days and ask people to rebase

Reply:
sgtm! nice catch